money.go

```diff
@@ -198,26 +198,56 @@ func (m *Money) Negative() *Money {
 }
 
 // Add returns new Money struct with value representing sum of Self and Other Money.
-func (m *Money) Add(om *Money) (*Money, error) {
-	if err := m.assertSameCurrency(om); err != nil {
-		return nil, err
+func (m *Money) Add(om ...*Money) (*Money, error) {
+	if len(om) == 0 {
+		panic("At least one Money is required to add")
 	}
 
-	return &Money{amount: mutate.calc.add(m.amount, om.amount), currency: m.currency}, nil
+	k := New(0, m.currency.Code)
+
+	for i := 0; i < len(om); i++ {
+		if err := m.assertSameCurrency(om[i]); err != nil {
+			return nil, err
+		}
+
+		k.amount = mutate.calc.add(k.amount, om[i].amount)
+	}
+
```

Comment:
You can use the alternative form:
```suggestion
	for _, m2:= range om {
		if err := m.assertSameCurrency(m2); err != nil {
			return nil, err
		}

		k.amount = mutate.calc.add(k.amount, m2.amount)
	}

```